.github/workflows/localstack.yml

```
@@ -0,0 +1,53 @@
+name: Deploy Domain Protect into LocalStack
```

This lengthy name will appear on each gha run as a prefix to the job name. Can we shorten it?

Currently this will be in gha with the format `{workflow_name} / {job_name}`

```
Deploy Domain Protect into LocalStack / localstack
```

Maybe something shorter like

```suggestion
name: LocalStack
```

And then we can rename the job to `Deploy `which then results in

```
LocalStack / Deploy
```